Directory.Build.props

```diff
@@ -4,7 +4,8 @@
     <Title>FsAutoComplete</Title>
     <Product>FsAutoComplete</Product>
     <PackageLicenseExpression Condition=" '$(PackAsTool)' != 'true' ">Apache-2.0</PackageLicenseExpression>
-    <NoWarn>3186,0042</NoWarn><!-- circumvent an error with the fake dependencymanager for paket: https://github.com/dotnet/fsharp/issues/8678 -->
+    <NoWarn>$(NoWarn);3186,0042</NoWarn><!-- circumvent an error with the fake dependencymanager for paket: https://github.com/dotnet/fsharp/issues/8678 -->
+    <NoWarn>$(NoWarn);NU1902</NoWarn><!-- NU1902 - package vulnerability detected -->
```

Comment:
Curious about this. What package is doing this?

Reply:
I'm on 8.0 SDKs for local development and the warning is coming from that. It's various NuGet.* packages that we have at version 6.3.0 and the vulnerability was addressed in....6.3.2 I think? I didn't want to fight with it right now so I just suppressed the warning.